sqe-http-api/DTOs/User.cs

```diff
@@ -22,4 +22,54 @@ public class LoginResponseDTO : UserDTO
     {
         public string token { get; set; }
     }
+    
+    public class NewUserDTO
+    {
+        /// <summary>
+        /// An object containing all data necessary to create a new user account
+        /// </summary>
+        /// <param name="userName">Short username for the new user account, must be unique</param>
+        /// <param name="email">Email address for the new user account, must be unique</param>
+        /// <param name="password">Password for the new user account</param>
+        /// <param name="organization">Name of affiliated organization (if any)</param>
+        /// <param name="forename">The user's given name (may be empty)</param>
+        /// <param name="surname">The user's family name (may be empty)</param>
+        public NewUserDTO(string userName, string email, string password, string organization, string forename, string surname)
+        {
+            this.userName = userName;
+            this.email = email;
+            this.password = password;
```

Comment:
Don't ever ever return passwords from the backend back to the frontend.

Reply:
This was a request DTO, not a response DTO.  The DTO has been renamed according to the comment below.

Done.